lightning/src/events/mod.rs

```diff
@@ -1234,7 +1245,7 @@ impl MaybeReadable for Event {
 					let user_channel_id = (user_channel_id_low_opt.unwrap_or(0) as u128) +
 						((user_channel_id_high_opt.unwrap_or(0) as u128) << 64);
 
-					Ok(Some(Event::ChannelClosed { channel_id, user_channel_id, reason: _init_tlv_based_struct_field!(reason, upgradable_required) }))
+					Ok(Some(Event::ChannelClosed { channel_id, user_channel_id, reason: _init_tlv_based_struct_field!(reason, upgradable_required), counterparty_node_id, channel_capacity }))
```

Comment:
nit: keep to 100chars per line

Reply:
Quick one. Is this including whitespaces before the code?

Reply:
I think so, we're not that precise about it generally